snuba/admin/static/components/query_display/query_display.tsx

```diff
@@ -0,0 +1,199 @@
+import React, { useEffect, useState } from "react";
+import Client from "../../api_client";
+import { Table } from "../../table";
+import { COLORS } from "../../theme";
+
+import { ClickhouseNodeData, QueryRequest, QueryResult } from "./types";
+
+type QueryState = Partial<QueryRequest>;
+
+function QueriesDisplay(props: {
```

Comment:
nit: can we make the class name QueryDisplay or the filename queries_display.tsx?